src/test/java/posters/pageobjects/pages/browsing/HomePage.java

```diff
@@ -17,9 +17,10 @@
 {
     @Override
     @Step("ensure this is a home page")
-    public void isExpectedPage()
+    public HomePage isExpectedPage()
     {
         $("#titleIndex").should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.